std_msgs/Header header
uint8[4] digital_array_AU8
int8[4] analogue_array_AI8
int16[2] analogue_array_AI16
int32 analogue_value_AI32
